docs/source/usage/cli.rst

```diff
@@ -126,17 +126,43 @@ The ``create-meta`` command also takes one optional argument:
 
 .. note::
 
-   The configuration file must be in the `.ini` format. For more information on the format, please refer to the `Python documentation <https://docs.python.org/3/library/configparser.html>`_.
+   The configuration file must be in the `.toml` format. For more information on the format, please refer to the `Python documentation <https://docs.python.org/3/library/configparser.html>`_.
 
    An example of a configuration file that specifies the ``PassengerId`` column to be unique and the ``Fare`` column to have a log-normal distribution is as follows:
 
-   .. code-block:: ini
+   .. code-block:: toml
 
-      [var.PassengerId]
-      unique = True
+      [[var]]
+      name = "PassengerId"
+      distribution = {unique = true}  # Notice lower capitalization for .toml files.
 
-      [var.Fare]
-      distribution=LogNormalDistribution
+
+      [[var]]
+      name = "Fare"
+      distribution = {implements = "core.log_normal"}
```

Comment:
Yes, it's possible, but do we want to advise our users to use that? If there is a "extra.log_normal" in a different distribution, then it will be selected depending on the order of the providers..

Reply:
makes sense to stick with core.log_normal